### PR TITLE
Fixed use statement in 07_Restricting_Public_Asset_Access.md

### PR DESCRIPTION
**Pull request by christianbader**
## Changes in this pull request  

Updated wrong use statement in `07_Restricting_Public_Asset_Access.md` and adapted function name according to the documentation.

### CODE REVIEW

File: doc/Development_Documentation/04_Assets/07_Restricting_Public_Asset_Access.md [hunk 1]
```diff
@@ -125,15 +125,16 @@ namespace App\Controller;
...
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
 use Symfony\Component\Routing\Matcher\UrlMatcher;
-use Symfony\Component\Routing\RouteCollection;use Symfony\Component\Templating\Storage\Storage;
+use Symfony\Component\Routing\RouteCollection;
 
 class MyAssetController extends FrontendController
 {
-    public function protectedAssetAction(Request $request)
+    public function protectedAsset(Request $request)
```
Comment (dvesh3) [inline doc/Development_Documentation/04_Assets/07_Restricting_Public_Asset_Access.md]:
```suggestion
    public function protectedAssetAction(Request $request)
```

Comment (christianbader) [reply doc/Development_Documentation/04_Assets/07_Restricting_Public_Asset_Access.md]: @dvesh3 Ok, so I'm also going to update the docs in line 116, I guess?

Comment (dvesh3) [reply doc/Development_Documentation/04_Assets/07_Restricting_Public_Asset_Access.md]: yes, please.

change it to: `App\Controller\MyAssetController:protectedAssetAction`

**Comment by dvesh3** [comment]
@christianbader thanks!

**Pull request merged**